### ISSUE TITLE
Issue with "Using your new knowledge": Change as Assessment?

### ISSUE THREAD

**Issue by glmvc** [mdn/content#11932]
MDN URL: https://developer.mozilla.org/en-US/docs/Learn/CSS/First_steps/Using_your_new_knowledge

#### What information was incorrect, unhelpful, or incomplete?
Shouldn't this page/article be categorized as an assessment?

#### What did you expect to see?
As with the other modules, that at the end there is (at least) one assessment and this page even looks like one, it's just not titled or linked that way etc....
Appropriate changes regarding this should of course also be made on the [overview page](https://developer.mozilla.org/en-US/docs/Learn/CSS/First_steps) of the module.

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/learn/css/first_steps/using_your_new_knowledge`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/CSS/First_steps/Using_your_new_knowledge
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/css/first_steps/using_your_new_knowledge/index.md
* Last commit: https://github.com/mdn/content/commit/07453ac2b659a397b0fee921fa71db5488e9ca62
* Document last modified: 2021-10-08T14:43:33.000Z

</details>

**Comment by wbamberg** [comment]
Yes, I agree, we should do this. Would you like to file a PR to do it?

**Comment by glmvc** [comment]
@wbamberg What needs to be taken into account? As I see the content is nearly the same as in the other assessments - so actually the **title should be adapted**.

What comes to my mind is to **add the assessment as a separate part on the overview page** of the module - i.e. with a heading and a short description.

By the way, I saw in the other assessments that there is a "**marking guide**". Should this and the associated (mdn/learning-area items) also be created in this case?

I didn't quite understand that with the marking guides, I actually only came across them via the discourse forum - in other words, nothing was mentioned about them in the learning modules/assessment pages (if I didn't read over it everywhere)... Is that the way it's supposed to be?

I apologize for the many questions, I'm still a beginner, but I would - as far as I can - do these things :)

**Comment by wbamberg** [comment]
OK, looking at this in more detail, it's quite complicated!

Minimally, yes:
* the page title and URL should be changed
* we should add an "Assessments" heading as we have for other modules, like https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks#assessments, and have this assessment under there.

And yes, we should probably also have a marking guide.

Also though, most of the supporting code for the learning area is in https://github.com/mdn/learning-area, but this assessment (and probably lots of the other CSS Learn stuff) is in https://github.com/mdn/css-examples/tree/main/learn/.So really that should all be moved, but that's probably a mini-project all on its own.

> I didn't quite understand that with the marking guides, I actually only came across them via the discourse forum - in other words, nothing was mentioned about them in the learning modules/assessment pages (if I didn't read over it everywhere)... Is that the way it's supposed to be?

Yes. The idea is that you don't assess yourself, but you ask in the forum for someone to assess you (see e.g. https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Image_gallery#assessment_or_further_help). Then the people who assess you (who are volunteers) use the marking guide.

We might one day want to reassess some of this, but it's not really a high priority at the moment.

I hope that helps!

**Comment by glmvc** [comment]
Thanks for the detailed explanation!
Oh so that makes sense with the assessments, yes - now I understand.
The things mentioned are on my to-do list. 👍

**Comment by wbamberg** [comment]
Actually I'm wrong about this bit:

> the page title and URL should be changed (https://github.com/mdn/content#moving-one-or-more-documents)

We only have to change the page title. See for example https://developer.mozilla.org/en-US/docs/Learn/Server-side/Django/django_assessment_blog, which has a title of "Assessment:  DIY Django mini blog" but whose URL doesn't have the "Assessment: " bit. So it's simpler, and we don't have to move the page.

**Comment by glmvc** [comment]
Changed with #13781
